prov/rxd/src/rxd_atomic.c

```diff
@@ -94,8 +94,8 @@ static struct rxd_x_entry *rxd_tx_entry_init_atomic(struct rxd_ep *ep, fi_addr_t
 			assert(len == tx_entry->bytes_done);
 		}
 	}
-	tx_entry->pkt->pkt_size = ((char *) ptr - (char *) base_hdr) +
-				ep->tx_prefix_size;
+
+	rxd_pkt_size(ep, tx_entry, base_hdr, ptr);
```

Comment:
My preference is for this to look like:
```
tx_entry->pkt->pkt_size = rxd_pkt_size(...)
```
That makes it clear that the pkt_size is being set, rather than hiding it inside a function.

Reply:
@shefty Ok. Done!